### ISSUE TITLE
am I doing something wrong?

### ISSUE THREAD

**Issue by U-Go-7** [cavi-au/Consent-O-Matic#39]
Hi, I just installed this because "Consent Blocker", which I use to stop google's redirect to their consent page, does not work on TechCrunch's Yahoo/Verizon powered cookies form.
Well, yours does not either.
Am I doing something wrong?

Other questions:
- does it reject also legitimate interest or only normal cookies?
- where we can add a new rule I saw a list of rules providers which can be added. Why do you provide this list instead of automatically include them all per default?
- just to be sure that I understand how it should work: if it works, I should not get a popup with the consent choice, right?

thanks

**Comment by NotLisztening** [comment]
Just wanted to leave a bump, because i have the exact same questions. "Legitimate interest" is the biggest "wtf" I have about cookies in general and I am tired of declining them.

**Comment by svnhub** [comment]
I don't think you are doing anything wrong. Noone has written a rule for the popup at TechCrunch yet, but I've reported it using the addon so it is on the list of what to look at now.

Legitimate interest: It depends on the Concent Management Provider (CMP) rule how normal cookies and legitimate interest are rejected. It is currently unclear from a legal perspective if "legitimate interest" is in fact a proper argument for them to default cookies to ON, but that is an entirely different discussion. Most of the newer rules will try to categorize the legitimate interest cookies and enable/disable them based on preferences just like the ordinary cookies. This is especially useful when they default to ON and you have manually click every single one of them...

Rules lists: There is one offical list (included with the addon by default) but people are free to publish their own lists of rules - either to expand on the official list without sending it in as a pull request or for testing purposes or other reasons.

Expected effect: When using Consent-O-Matic on a supported page (one that uses one of the known CMP popups) you should only momentarily be able to see the addon badge at the top of the screen as it clicks all the buttons to fill out the popup with your preferences. If something fails along the line it will typically leave the popup open for you to manually fill it out - in that case you can report the site using the addon logo.
